src/gui/plugins/component_inspector/ComponentInspector.qml

```diff
@@ -216,6 +216,26 @@ Rectangle {
         }
       }
 
+      ToolButton {
+        id: addSystemButton
+        checkable: false
+        text: "\u002B"
+        contentItem: Text {
+          text: addSystemButton.text
+          color: "#b5b5b5"
+          horizontalAlignment: Text.AlignHCenter
+          verticalAlignment: Text.AlignVCenter
+        }
+        visible: entityType == "model"
+        ToolTip.text: "Add a system to a model"
```

Comment:
To be more generic when we enable other entity types, maybe

```suggestion
        ToolTip.text: "Add a system to this entity"
```

Reply:
[da4f16a](https://github.com/gazebosim/gz-sim/pull/1549/commits/da4f16a0067dcce3bd0ae96d543786058646effc)